docs/design/uptime-reputation-calculation.md

```diff
@@ -0,0 +1,177 @@
+# Uptime reputation caculation
+
+## Abstract
+
+Having a fair, simple and understandable way to calculate the uptime of _storagenodes (henceforth SNs)_ is crucial for not causing unfair disqualifications, clarifying how _storagenode operators (henceforth SNOs)_ should administrate and maintain their _SN_ for being align with the network requirements and last but not least have a clear proof of disqualification for _SN_ getting disqualified due to too many uptime check failures.
```

Comment:
Let's not use abbreviations that aren't in the common vocabulary of software development. These design documents are intended to be read by other people, not just the development team.

Reply:
All the abbreviations are mentioned what they are before using them despite that they belong to the Storj business domain.
I've seen other documents using the abbreviations without explaining what they are.